packit_service/worker/allowlist.py

```diff
@@ -0,0 +1,346 @@
+        return False
+
+    def add_account(self, account_login: str, sender_login: str) -> bool:
+        """
+        Add account to allowlist.
+        Status is set to 'waiting' or to 'approved_automatically'
+        if the account is a packager in Fedora.
+        :param sender_login: login of the user who installed the app into 'account'
+        :param account_login: login of the account into which the app was installed
+        :return: was the account (auto/already)-allowlisted?
+        """
+        # TODO: Switch to AllowlistModel
+        if AllowlistModel.get_account(account_login):
+            return True
+
+        # TODO: Switch to AllowlistStatus
+        AllowlistModel.add_account(account_login, AllowlistStatus.waiting.value)
+
+        if self._signed_fpca(sender_login):
+            AllowlistModel.add_account(
+                account_login, AllowlistStatus.approved_automatically.value
+            )
+            return True
+
+        return False
+
+    @staticmethod
+    def approve_account(account_name: str):
+        """
+        Approve user manually
+        :param account_name: account name for approval
+        """
+        AllowlistModel.add_account(
+            account_name=account_name, status=AllowlistStatus.approved_manually.value
+        )
+
+        logger.info(f"Account {account_name!r} approved successfully.")
+
+    @staticmethod
+    def is_approved(account_name: str) -> bool:
+        """
+        Check if user is approved in the allowlist
+        :param account_name: account name to check
+        :return:
+        """
+        account = AllowlistModel.get_account(account_name)
+        if account:
+            db_status = account.status
+            s = AllowlistStatus(db_status)
+            return (
+                s == AllowlistStatus.approved_automatically
+                or s == AllowlistStatus.approved_manually
+            )
+
+        return False
+
+    @staticmethod
+    def remove_account(account_name: str) -> bool:
+        """
+        Remove account from allowlist.
+        :param account_name: account name for removing
+        :return: has the account existed before?
+        """
+        account_existed = False
+
+        if AllowlistModel.get_account(account_name):
+            AllowlistModel.remove_account(account_name)
+            logger.info(f"Account {account_name!r} removed from postgres allowlist!")
+            account_existed = True
+
+        if not account_existed:
+            logger.info(f"Account {account_name!r} does not exist!")
+
+        return account_existed
+
+    @staticmethod
+    def accounts_waiting() -> List[str]:
+        """
+        Get accounts waiting for approval
+        :return: list of accounts waiting for approval
+        """
+        return [
+            account.account_name
+            for account in AllowlistModel.get_accounts_by_status(
+                AllowlistStatus.waiting.value
+            )
+        ]
+
+    def unchecked_event(
```

Comment:
What about some prefix for those methods? Something like `validate_unchecked_event`, `validate_release_push_event`,...